google-cloud-language/lib/google/cloud/language/v1/doc/google/cloud/language/v1/language_service.rb

```diff
@@ -119,28 +124,53 @@ module Type
             # Event
             EVENT = 4
 
-            # Work of art
+            # Artwork
             WORK_OF_ART = 5
 
-            # Consumer goods
+            # Consumer product
             CONSUMER_GOOD = 6
 
-            # Other types
+            # Other types of entities
             OTHER = 7
 
-            # Phone number
+            # Phone number<br><br>
```

Comment:
@dazuma Is inline HTML acceptable here? Or is this a  code generation mistake?

Reply:
It's intentional. It comes from a change in the proto comments: https://github.com/googleapis/googleapis/commit/233e53488c497e6e3aba6ff9b7f14b9839128451#diff-3585a7188a84bb404ea0593aaaa5f5e6R192